cleanlab/outlier.py

```diff
@@ -0,0 +1,545 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
```

Comment:
Leaving comment here that actually applies to the `outliers.ipynb` notebook instead (but github won't let me comment there due to diff size):

Rename "How does cleanlab detect outliers?"
-> "How does cleanlab detect outliers from feature values?"
and move this section up to the end of the features-based OOD detection part of the tutorial, before you launch into pred_probs based OOD detection.